src/service/genesis_gen.h

```diff
@@ -420,6 +420,13 @@ namespace ccf
         {.status = CodeStatus::ALLOWED_TO_JOIN, .platform = platform});
     }
 
+    void trust_node_security_policy(
+      const RawPolicy& raw_security_policy, const DigestedPolicy& host_data)
+    {
```

Comment:
This needs to be exposed via the JS bindings, and then to governance through an action. We need a test, and a documentation sample illustrating how to use it.

Reply:
Added some docs, testing bad security policies has some coverage of the actions since they rely on us removing the default policy and re-adding it later. I think that's sufficient if you agree?